2.8/debian-10/rootfs/opt/bitnami/scripts/libkafka.sh

```diff
@@ -683,6 +709,30 @@ kafka_configure_producer_consumer_message_sizes() {
     fi
 }
 
+########################
+# Initialize KRaft
+# Globals:
+#   KAFKA_*
+# Arguments:
+#   None
+# Returns:
+#   None
+#########################
+kraft_initialize(){
+  info "Initializing KRaft..."
+
+  if [[ -z "${KAFKA_CLUSTER_ID:-}" ]]; then
+    warn "KAFKA_CLUSTER_ID not set - If using multiple nodes then you must use the same Cluster ID for each one"
+    info "Generating Cluster ID"
+    KAFKA_CLUSTER_ID=$($KAFKA_HOME/bin/kafka-storage.sh random-uuid)
+    info "KAFKA_CLUSTER_ID: ${KAFKA_CLUSTER_ID}"
+    export KAFKA_CLUSTER_ID="${KAFKA_CLUSTER_ID}"
+  fi
```

Comment:
Minor improvements (also the `export` should not be necessary, since it was already exported in `kafka-env.sh`):

```suggestion
  if [[ -z "$KAFKA_CLUSTER_ID" ]]; then
    warn "KAFKA_CLUSTER_ID not set - If using multiple nodes then you must use the same Cluster ID for each one"
    KAFKA_CLUSTER_ID="$($KAFKA_HOME/bin/kafka-storage.sh random-uuid)"
    debug "Generated Kafka cluster ID '${KAFKA_CLUSTER_ID}'"
  fi
```